.github/workflows/build.yaml

```
@@ -88,8 +88,10 @@ jobs:
         run: |
           git config --global user.email "TravisCI"
           git config --global user.name "Automated Deployment"
-      
+
       - name: Script
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

`run` steps don't have `with` property, you need to add this to env variables:
```suggestion
        env:
          github_token: ${{ secrets.GITHUB_TOKEN }}
```
Are you using github cli?

---

got it, thank you!